GitCommands/IDiffListSortService.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reactive.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace GitCommands
+{
+    public enum DiffListSortType
+    {
+        /// <summary>
+        /// Invalid default
+        /// </summary>
+        Invalid,
```

Comment:
I would remove this, and make `FilePath = 0`

Reply:
Will do